src/components/SearchBar/searchBar.component.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import './searchBar.styles.css';
+
+//icons
+import searchIcon from '../../assets/search-icon.svg';
+
+import { connect } from 'react-redux';
+import {
+  toggleSearchbar,
+  searchProducts,
+  searchStateToggle,
+} from '../../actions/search';
+
+class SearchBar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchInput: '',
+    };
+  }
+
+  onChange = (event) => {
+    const { name, value } = event.target;
+    this.setState({ [name]: value });
+  };
+
+  onSubmit = (event) => {
+    event.preventDefault();
+    const { searchInput } = this.state;
+    const { searchProducts, searchStateToggle } = this.props;
+    searchProducts(searchInput);
+    searchStateToggle();
+  };
+
+  initSearch = (event) => {
+    const { toggleSearchbar } = this.props;
+    this.onSubmit(event);
+    toggleSearchbar();
+  };
+
+  render() {
+    const { toggleSearchbar, displaySearchbar } = this.props;
+
+    return (
+      <div className="navbar-user-utils">
+        {displaySearchbar ? (
+          <>
```

Comment:
Change this to <React.Fragment></React.Fragment> instead. It is more descriptive than <></>